spec/requests/providers/means/employed_incomes_spec.rb

```diff
@@ -86,25 +86,12 @@
         context "when the application is using the bank upload journey" do
           let(:application) { create(:legal_aid_application, provider_received_citizen_consent: false) }
 
-          before do
-            permission = create(:permission, :bank_statement_upload)
-            provider.permissions << permission
-            provider.save!
-          end
-
           it "redirects to the regular income page" do
             request
             expect(response).to redirect_to(providers_legal_aid_application_means_regular_incomes_path(application))
           end
         end
 
-        context "when the application is not using the bank upload journey" do
```

Comment:
is this path/flow not ppossible now for truelayer? 🤷

Reply:
Heavy handed pruning! 🤷‍♂️